ouroboros-consensus/src/Ouroboros/Consensus/Mempool/Impl.hs

```diff
@@ -442,6 +437,13 @@ implGetSnapshotFor MempoolEnv{mpEnvStateVar, mpEnvLedgerCfg}
         . internalStateFromVR
         . validateStateFor mpEnvLedgerCfg blockSlot ledger
 
+-- TODO #1498:
```

Comment:
Can you make this a proper haddock docstring:

```haskell
-- | Return the current capacity of the Mempool.
--
-- TODO #1498: As of right now, ...
```

Reply:
Done